framework/src/modules/chain/transport/transport.js

```diff
@@ -363,39 +363,24 @@ class Transport {
 					query,
 				},
 			);
-			throw new Error(errors);
+			throw errors;
 		}
 
 		let block;
-		let success = true;
-		try {
-			block = blocksUtils.addBlockProperties(query.block);
+		block = blocksUtils.addBlockProperties(query.block);
```

Comment:
why not just `let block = blocksUtils.addBlockProperties(query.block);`?

Reply:
I would even do the `const block`, but since this is patch case and not to change a lot (because there is alot of those cases in this file), i would prefer not to touch it now

Reply:
ah nvm i think it's mistake will fix it